boards/st/nucleo_h533re/doc/index.rst

```diff
@@ -0,0 +1,306 @@
++===========+============+=====================================+
+| CLOCK     | on-chip    | reset and clock control             |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| PWM       | on-chip    | PWM                                 |
++-----------+------------+-------------------------------------+
+| RNG       | on-chip    | True Random number generator        |
++-----------+------------+-------------------------------------+
+| RTC       | on-chip    | Real Time Clock                     |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| WATCHDOG  | on-chip    | independent watchdog                |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not yet supported on this Zephyr port.
+
+The default configuration can be found in the defconfig and dts files:
+
+- Secure target:
+
+  - :zephyr_file:`boards/st/nucleo_h533re/nucleo_h533re_defconfig`
+  - :zephyr_file:`boards/st/nucleo_h533re/nucleo_h533re.dts`
+
+Zephyr board options
+====================
+
+The STM32H533 is a SoC with Cortex-M33 architecture. Zephyr provides support
+for building for Secure firmware.
+
+The BOARD options are summarized below:
+
++----------------------+-----------------------------------------------+
+|   BOARD              | Description                                   |
++======================+===============================================+
+| nucleo_h533re        | For building Secure firmware                  |
++----------------------+-----------------------------------------------+
+
+Connections and IOs
+===================
+
+Nucleo H533RE Board has 8 GPIO controllers. These controllers are responsible for pin muxing,
+input/output, pull-up, etc.
+
+For more details please refer to `STM32H5 Nucleo-64 board User Manual`_.
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+
+- ADC1 channel 14 input: PB1
+- USART1 TX/RX : PB14/PB15 (Arduino USART1)
+- SPI1 SCK/MISO/MOSI/NSS: PA5/PA6/PA7/PC9
+- UART2 TX/RX : PA2/PA3 (VCP)
+- USER_PB : PC13
+
+System Clock
+------------
+
+Nucleo H533RE System Clock could be driven by internal or external oscillator,
+as well as main PLL clock. By default System clock is driven by PLL clock at
+240MHz, driven by 8MHz external clock provided from the STLINK-V3EC.
```

Comment:
There is a 24MHz (X3) for the HSE external clock on the nucleo_h533 board
--> HSE is not STLINK sourced